test/release/examples/primers/arrayVectorOps.chpl

```diff
@@ -1,5 +1,18 @@
 // Array Vector Operations
 
+//
+// **DEPRECATION WARNING**
+//
+// Vector operations on arrays have been deprecated for the 1.20 release, and
```

Comment:
How about we emphasize that it will be removed eventually, and provide a link to the new module, e.g.

```chpl
// Vector operations on arrays have been deprecated for the 1.20 release, and
// will be removed in future releases. The  ``list`` from the  :mod:`List` module 
// has been added to the standard library, and is intended to ...
```